contributing-guides/style-guide.md

```diff
@@ -373,10 +373,7 @@ It should instead be simplified to make it easier for everyone to read:
 
 - Proper names should be capitalized in the description whenever applicable (e.g. use `A tool for interacting with a Git repository.` instead of ``A tool for interacting with a `git` repository.``).
 - Acronym expansions (i.e. protocols, tools, etc) must not be translated unless there is a recognized native equivalent for them.
-- When documenting keycaps or a keyboard shortcut for a utility, make it stand out in the description:
-
-1. If it is not translatable, enclose it with backticks (i.e. ``Print the last lines of a given file and keep reading it until `Ctrl + C`:``)
-2. If it is translatable, enclose it with double angled brackets inside a placeholder (i.e. ``:wq{{<<Enter>>}}``).
+- When documenting keycaps or a keyboard shortcut for a utility, make it stand out in the description (i.e. ``Print the last lines of a given file and keep reading it until `Ctrl + C`:``).
```

Comment:
This should be reformatted.
I tested with `grep -r "<<Enter>>"`. This pattern is used nowhere else in the whole project.

Reply:
See https://github.com/Managor/tldr/blob/style-guide-10/pages/linux/quickemu.md for a place where this needs to be standardised

Reply:
> This should be reformatted. I tested with `grep -r "<<Enter>>"`. This pattern is used nowhere else in the whole project.

Sure

Reply:
What do you think, should we use the same syntax in the description as we use in the examples? It would require making sure that they're always in backticks so that they're not invisible to Github markdown rendering, but the consistency is nice for intuitiveness.